images/installer/nested-libvirt/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# This Dockerfile is a used by CI to test a libvirt cluster launched in a gce instance
```

Comment:
`images/installer` is not a great name for distinguishing among several images, because all of the images in this repo are likely to relate to the installer ;).  What it's getting at though, is "just the installer".  Can we move this new libvirt-targetting image into `images/nested-libvirt`?

Reply:
yes, moved